0020-hnt-max-supply.md

```diff
@@ -21,11 +21,13 @@ With the introduction of a hard cap on HNT supply, Helium’s token-economics wo
 
 # Stakeholders
 
-All HNT holders, Hotspot owners, and HST holders will be affected by this HIP. Users of the network will see no change to data transmission pricing.
+All HNT holders, Hotspot owners, and HST holders will be affected by this HIP. 
+
+There will be no change to the cost to transfer data because the cost of Data Credits is fixed in USD terms.
 
 # Detailed Explanation
 
-The proposal is to have halvenings of net HNT issuance every 2 years on the anniversary of genesis. This means the first halvening will be on 8/1/2021 and net HNT issuance will be reduced to 2.5M HNT per month.
+The proposal is to have halvenings of net HNT issuance every 2 years on the anniversary of genesis. This means the first halvening will be on August 1st, 2020 and net HNT issuance will be reduced to 2.5M HNT per month.
```

Comment:
2020 or 2021?

Reply:
2021, thank you for catching the typo. fixed it now